src/cmds/versions/create.js

```diff
@@ -71,19 +70,48 @@ module.exports = class CreateVersionCommand {
       );
     }
 
-    if (!fork) {
+    let versionList;
+    if (!opts.fork) {
       versionList = await fetch(`${config.get('host')}/api/v1/version`, {
         method: 'get',
         headers: cleanHeaders(key),
       }).then(res => handleRes(res));
     }
 
-    const versionPrompt = promptOpts.createVersionPrompt(versionList || [{}], {
-      newVersion: version,
-      ...opts,
-    });
+    const questions = [
+      {
+        type: 'select',
+        name: 'fork',
+        message: 'Which version would you like to fork from?',
+        skip: () => opts.fork,
+        choices: (versionList || [{}]).map(v => ({ message: v.version, value: v.version })),
+      },
+      {
+        type: 'confirm',
+        name: 'main',
+        message: 'Would you like to make this version the main version for this project?',
```

Comment:
@kanadgupta Thoughts on slightly reworking this to the following? The double "version" in there reads off.

```suggestion
        message: 'Would you like to make this the main version for this project?',
```

Reply:
Agreed! Nice find